Dockerfile

```diff
@@ -115,6 +146,15 @@ ARG PACKAGES
 RUN apt-get install -y --no-install-recommends ca-certificates tzdata libunwind8 $PACKAGES \
     && rm -rf /var/lib/apt/lists/*
 
+# Install GPU dependencies
+ARG GPU
+RUN if [ -n "$GPU" ]; then \
+    apt-get update && apt-get install -y \
+    libvulkan1 \
+    libvulkan-dev \
+    vulkan-tools \
+    ; fi
```

Comment:
I'd restructure this block like this. This way it does not trigger `apt-get update` twice, and have most reasonable caching order.

```
RUN apt-get update

# Install GPU dependencies
ARG GPU

RUN [ -n "$GPU" ] && apt-get install -y libvulkan1 libvulkan-dev vulkan-tools

# Install additional packages into the container.
# E.g., the debugger of choice: gdb/gdbserver/lldb.
ARG PACKAGES

RUN apt-get install -y --no-install-recommends ca-certificates tzdata libunwind8 $PACKAGES \
	&& rm -rf /var/lib/apt/lists/*
```

Reply:
`[ -n "$GPU"] && ...` doesn't work in the Dockerfile... 🙄

I guess, the next best thing would be
```
RUN if [ -n "$GPU" ]; then \
        apt-get install -y libvulkan1 libvulkan-dev vulkan-tools; \
    fi
```

Sorry for misleading!